src/Main.elm

```diff
@@ -27,13 +45,21 @@ init =
 
 type Msg
     = SquareClicked Int
+    | LayTile Int
```

Comment:
Some mock actions. Now the square can be selected, which then shows the two buttons within it to lay or cancel.